src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -118,58 +118,58 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AvoidAsyncVoidTitle" xml:space="preserve">
-    <value>Avoid Async Void</value>
+    <value>Avoid 'async void'.</value>
```

Comment:
A few of the manual fixes, like this one for example, wouldn't be caught by an analyzer.

Reply:
A lot of entries in the form `Axxx Bxxx Cxxx` could be relatively easily caught. 

Regarding quoting keywords, we could find a way, although it might be less elegant.

I am more concerned about types and properties.